client/routes/RouteComponents.jsx

```diff
@@ -63,3 +63,25 @@ InterceptAnonymizableRoute.propTypes = {
         PropTypes.node
     ]).isRequired
 };
+
+export const RedirectRouteForActiveSession = ({ children, ...rest }) => {
+    return (
+        <Route
+            {...rest}
+            render={props => {
+                if (!Session.isSessionActive()) {
+                    return children;
+                }
+
+                location.href = '/';
```

Comment:
When we're already logged in, don't let users reach the "/login/new" or "/login/create-account" view